src/fastcs_pandablocks/panda/panda_controller.py

```diff
@@ -1,14 +1,34 @@
 import asyncio
+from dataclasses import dataclass
+from typing import Any
 
-from fastcs.attributes import Attribute, AttrR, AttrRW, AttrW
-from fastcs.controller import Controller
-from fastcs.cs_methods import Scan
+from fastcs.attributes import Attribute, AttrR
+from fastcs.connections import IPConnectionSettings
+from fastcs.controllers import Controller
+from fastcs.datatypes import Table
+from fastcs.logging import bind_logger
+from fastcs.methods import Scan
+from pandablocks.utils import words_to_table
 
 from fastcs_pandablocks.panda.blocks import Blocks
 from fastcs_pandablocks.panda.client_wrapper import RawPanda
+from fastcs_pandablocks.panda.io import (
+    ArmIO,
+    DefaultFieldIO,
+    TableFieldIO,
+    TableFieldIORef,
+    UnitsIO,
+    panda_value_to_attribute_value,
+)
 from fastcs_pandablocks.types import PandaName
 
-from .handlers import panda_value_to_attribute_value
+logger = bind_logger(__name__)
+
+
+@dataclass
+class PandaControllerSettings:
+    poll_period: float
+    ip_settings: IPConnectionSettings
 
 
 class PandaController(Controller):
```

Comment:
I am a bit confused why `_update` isn't just decorated with `@scan`. Maybe because it
wasn't possible at the time?